sdk/python/kfp/v2/compiler/compiler_utils.py

```diff
@@ -52,3 +53,40 @@ def to_message(param: dsl.PipelineParam):
     return result
 
   return {param.name: to_message(param) for param in pipeline_params}
+
+
+def build_runtime_config_spec(
+    pipeline_root: Optional[str] = None,
+    pipeline_parameters: Optional[Mapping[str, Any]] = None,
+) -> pipeline_spec_pb2.PipelineJob.RuntimeConfig:
+  """Converts pipeine parameters to runtime parameters mapping.
+
+  Args:
+    pipeline_root: The root of pipeline outputs. Optional.
+    pipeline_parameters: The mapping from parameter names to values. Optional.
+
+  Returns:
+    A pipeline job RuntimeConfig object.
+  """
+
+  def _get_value(
+      value: Union[int, float, str]) -> Optional[pipeline_spec_pb2.Value]:
```

Comment:
Given `None` is allowed in the function, shall we do `Optional[Union[int, float, str]]` here?

Reply:
Good catch!